tests/integration/test_basket_waitlist_subscription.py

```diff
@@ -217,18 +229,30 @@ def check_subscribed():
         return details
 
     contact_details = check_subscribed()
-    # CTMS has both newsletters.
-    newsletters_by_name = {nl["name"]: nl for nl in contact_details["newsletters"]}
-    assert "relay-waitlist" in newsletters_by_name
-    assert waitlist in newsletters_by_name
-    # Country is taken from one of them.
+    assert contact_details["newsletters"] == []
+    assert contact_details["waitlists"] == [
+        {
+            "name": "relay",
+            "source": "https://relay.firefox.com/",
+            "fields": {
+                "geo": "es",
+            },
+        },
+        {
+            "name": "relay-vpn-bundle",
+            "source": "https://relay.firefox.com/vpn-relay/waitlist/",
+            "fields": {
+                "geo": "fr",
+            },
+        },
+    ]
     assert contact_details["relay_waitlist"] == {
-        "geo": "fr",
+        "geo": "es",
```

Comment:
Shall we add a comment here that it takes the last one of the list here? https://github.com/mozilla-it/ctms-api/blob/23bf39231ab680a27f2518cee4f47186602fa28b/ctms/schemas/contact.py#L235

Reply:
Yes -- I think I was hesitant because I wasn't sure that it'd always be the last.

Reply:
Separately, Should we also modify this behavior in the future to make it so that the country is taken from the waitlist that was most recently updated?

Reply:
Wait actually, now I'm confused again. We expect that it'd be the last one of the list, but that assertion is saying that the country is `"es"`, when the "last one" is `"fr"`.

Reply:
With the changes of this PR, it will take the first one. I'm sorry about the confusion